db/seeds.rb

```diff
@@ -6,6 +6,9 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+puts("Seed start...")
 Destination.create(name: 'Bora Bora', location: 'French Polynesia', image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636638534/bora-bora-island_tcyoev.jpg', price_per_day: 150)
 Destination.create(name: 'Rome', location: 'Italy', image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636638500/Rome-Italy_isahvk.jpg',price_per_day: 100)
 Destination.create(name: 'Gracier National Park', location: 'USA', image_url: 'https://res.cloudinary.com/dqdrsilxz/image/upload/v1636573679/sample.jpg',price_per_day: 100)
+User.create(email: 'user@user', password:'123456')
+puts("Seed finished...")
```

Comment:
[Optional]
To create user we should provide email, name/username,   password, and password_confirmation.